source/MaterialXView/Material.cpp

```diff
@@ -179,6 +179,79 @@ void Material::updateUniformsList()
     delete[] uniformName;
 }
 
+std::unordered_map<std::string, std::string> Material::uniformValues() const
```

Comment:
I don't understand why the actual hardware shader needs to be scanned as there is a Shader instance which is kept which should have all the current values.

Reply:
I tried just getting from `getPublicUniforms` but the changes were not there, let me check again. thanks